src/api/api.ts

```diff
@@ -5,6 +5,8 @@ import type {Channel} from '../records/channel';
 import type {Role} from '../records/role';
 import type {Embed} from '../components/embed';
 
+export const GatewayURI = 'https://discord.com/api/v9'
```

Comment:
Should we call this `API_URL` instead, since we aren't planning on using the Gateway?